### ISSUE TITLE
Deep Learning insights: Lessons learned from BUI imputation

### ISSUE THREAD

**Issue by dylanhatlas** [dylanhatlas/Colab#1]
- Using large batch size (~50% of training dataset) smoothed out learning process, otherwise losses fluctuated dramatically with each step
- Scale target variable with large range (0-millions) to enable consistent learning. In this case divide by 1e5. Note that this discounts smaller values.
- Validation not working properly
- Consider exploring adaptive learning rate and smaller batch sizes on other tasks
-
